### PR TITLE
Two MCD logging patches

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by cgwalters** [comment]
Since this bounced I added another logging patch.

File: pkg/daemon/node.go [hunk 1]
```diff
@@ -40,6 +41,7 @@ func loadNodeAnnotations(client corev1.NodeInterface, node string) error {
 		return fmt.Errorf("Failed to unmarshal initial annotations: %v", err)
 	}
 
+	glog.Infof("Setting initial node config: %s", initial[CurrentMachineConfigAnnotationKey])
```
Comment (ashcrow) [inline pkg/daemon/node.go]: Note: If the annotation isn't set for some reason this will output empty.

**Comment by ashcrow** [comment]
Unrelated failures

/retest

**Pull request merged**